### PR TITLE
[RW-7364] repro the flakytest

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Comment by jmthibault79** [comment]
@calbach TBH I'm no longer sure this is a legitimate repro.  It could easily be true that *this* change is causing its own *different* failures.

File: ui/src/app/pages/homepage/data-access-requirements.spec.tsx [hunk 1]
```diff
@@ -278,9 +278,15 @@ describe('DataAccessRequirements', () => {
...
     });
 
+    const repeat = async(n, callback) => await Promise.all(Array(n).fill(0).map(callback));
+    const TEST_ITERATIONS = 1;  // 1 passes, 2 fails
+    let currentIteration = 0;
+
     // RAS launch bug (no JIRA ticket)
     // temporarily disabled for flakiness - https://precisionmedicineinitiative.atlassian.net/browse/RW-7364
-    xit('should render all modules as complete by transitioning to all complete', async() => {
+    it('should render all modules as complete by transitioning to all complete', async() => repeat(TEST_ITERATIONS, async() => {
```
Comment (calbach) [inline ui/src/app/pages/homepage/data-access-requirements.spec.tsx:287]: When I run this locally, I see many errors about updates being applied while unmounted. There seem to be a lot of issues with writing this kind of dynamically updating test case which involve direct store interactions.

One way of dealing with this is to use fake jest timers. An example of that is here: runtime-utils.spec.tsx . It's possible this resolves the issue, but again - I haven't been able to repro so it's hard to say.

**Review by calbach — COMMENTED** [review]
So another issue I realized when trying to reproduce this issue is that the test order is based on several heuristic factors, such as last failed and most recently observed test speed.

You can force a deterministic order, which I tried by forcing the same order observed during your test case: https://github.com/all-of-us/workbench/compare/ch/repro-jest-flake

Unfortunately I still failed to reproduce using this approach.

File: ui/src/app/pages/homepage/data-access-requirements.spec.tsx [hunk 2]
```diff
@@ -278,9 +278,15 @@ describe('DataAccessRequirements', () => {
         expect(findCompletionBanner(wrapper).exists()).toBeTruthy();
     });
 
+    const repeat = async(n, callback) => await Promise.all(Array(n).fill(0).map(callback));
```
[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/pages/homepage/data-access-requirements.spec.tsx:281]: Could you paste the relevant portion in?  This link didn't resolve to a specific location for me.

[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/pages/homepage/data-access-requirements.spec.tsx:281]: Also, changing to sequential repetition caused it to no longer fail: https://github.com/all-of-us/workbench/pull/5698

File: ui/src/app/pages/homepage/data-access-requirements.spec.tsx (hunk 1 above)
Comment (jmthibault79) [reply ui/src/app/pages/homepage/data-access-requirements.spec.tsx:287]: OK, let's give it a try at #5699

**Comment by jmthibault79** [comment]
Closing because #5699 seems to have resolved this.

**Pull request closed without merging**